File: baekjoon/bronze/level1/test_11050/java/JunitTest.java
package bronze.level1.test_11050.java;

import org.junit.Test;
import util.TestUtil;

import java.io.IOException;

import static org.junit.Assert.assertEquals;
import static util.ThrowingRunnable.runUnchecked;

public class JunitTest {

    @Test
    public void test_1() throws IOException {
        String input = "5 2";
        String expectedOutput = "10";

        String actualOutput = TestUtil.executeTest(input, () -> runUnchecked(() -> Main.main(new String[0])));
        assertEquals(expectedOutput, actualOutput);
    }

}

File: baekjoon/bronze/level1/test_1934/kotlin/JunitTest.kt
package bronze.level1.test_1934.kotlin;

import junit.framework.TestCase.assertEquals
import org.junit.Test
import util.TestUtil

class JunitTest {

    @Test
    fun test1() {
        val input = """
            3
            1 45000
            6 10
            13 17
        """.trimIndent()
        val expectedOutput = """
            45000
            30
            221
            
        """.trimIndent()

        val actualOutput = TestUtil.executeTest(input) { main() }
        assertEquals(expectedOutput, actualOutput)
    }

}
